File: 800/watermelon/watermelon.py
# https://codeforces.com/problemset/problem/4/A
def watermelon(n):
  if n < 1 or n > 100:  # check if the weight is valid or invalid
    print("Invalid.")
  elif n >= 1 and n <= 100:
    if n % 2 == 0:      # check if the weight is even number
      print("Yes.")
    else:
      print("No.")

n = int(input(()))
watermelon(n)  

File: 800/watermelon/watermelon.c
// 

#include <stdio.h>

int main() {
    // Write C code here
    int w;
    printf("Enter the weight of the watermelon: ");
    scanf("%d", &w);
    printf("Weight is: %d", w);
    printf("\n");
    if(w >= 1 && w <= 100) {
        if(w % 2 == 0) {
            printf("YES.\n");
        }
        else {
            printf("NO.\n");
        }
    }
    return 0;
}
